tests/test_store.py

```diff
@@ -70,3 +82,27 @@ def test_autogen(self):
         with self.assertRaises(CSStoreError):
             # different key causes MAC error during decryption
             self.assertEqual(enc2.get('key'), 'value')
+
+    def test_secret_protection_default(self):
+        enc = EncryptedOverlay(self.parser, 'store:enc_default')
+        enc.store = self.backing_store
+        key = 'key1'
+        enc.set(key, 'value1')
+        self.assertEqual(enc.protected_header['enc'], enc.master_enctype)
+        self.assertNotIn('custodia.key', enc.protected_header)
+        self.assertEqual(enc.secret_protection, 'encrypt')
+        self.assertEqual(enc.get(key), 'value1')
+        with self.assertRaises(CSStoreError):
+            self.assertEqual(enc.get('key'), 'value')
```

Comment:
don't think we need this test here, we are already testing above in test_autogen